sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/DynamicPruning.scala

```diff
@@ -37,24 +37,24 @@ trait DynamicPruning extends Predicate
  *  can reuse the results of the broadcast through ReuseExchange
  * @param broadcastKeyIndex the index of the filtering key collected from the broadcast
  */
-case class DynamicPruningSubquery(
+case class PartitionPruningSubquery(
     pruningKey: Expression,
     buildQuery: LogicalPlan,
     buildKeys: Seq[Expression],
     broadcastKeyIndex: Int,
     onlyInBroadcast: Boolean,
     exprId: ExprId = NamedExpression.newExprId)
   extends SubqueryExpression(buildQuery, Seq(pruningKey), exprId)
-  with DynamicPruning
-  with Unevaluable {
+    with DynamicPruning
+    with Unevaluable {
```

Comment:
The original indentation is correct.
- https://github.com/databricks/scala-style-guide/blob/master/README.md#indent